src/main/java/com/minecolonies/coremod/inventory/GuiHandler.java

```diff
@@ -53,7 +53,6 @@ else if (id == ID.CITIZEN_INVENTORY.ordinal())
 
             return new ContainerMinecoloniesCitizenInventory(player.inventory,
                                                               citizen.getCitizenEntity().getInventoryCitizen(),
-                                                              player,
                                                               colony.getID(),
                                                               building.getID(),
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') A "NullPointerException" could be thrown; "building" is nullable here. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS2259)